src/views/index.html

```diff
@@ -8,7 +8,7 @@
     <meta name="description" content="">
     <meta name="author" content="">
     <link rel="icon" href="../../favicon.ico">
-    <title>Autographa Lite</title>
+    <title>Autographa Live</title>
```

Comment:
This is Lite.

Reply:
I have removed this file and added the contents of this HTML to index.html inside public folder since we cannot alter the path...